elements/table.js

```diff
@@ -196,12 +194,15 @@ function tableElement (dats, send) {
 // ([obj], fn) -> html
 function createTable (dats, send) {
   return dats.map(dat => {
-    const progress = dat.stats.blocksTotal
-      ? Math.round(dat.stats.blocksProgress / dat.stats.blocksTotal * 100)
+    const stats = dat.stats && dat.stats.get()
+    const progress = (stats)
```

Comment:
My main point with style is just that it should be consistent over the whole project. Either stick to the existing style, or make the rest of the code base follow along in an atomic step.